### PR TITLE
Fix name of deprecated oc_requesttoken

### PR DESCRIPTION
**Pull request by ecsecta**
oc_requesttoken is marked as deprecated.

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @juliushaertl @skjnldsv @ChristophWurst

File: developer_manual/basics/front-end/js.rst [hunk 1]
```diff
@@ -34,14 +34,14 @@ listen to the ``OCA\Files::loadAdditionalScripts`` event:
 Sending the CSRF token
 ----------------------
 
-If any other JavaScript request library than jQuery is being used, the requests need to send the CSRF token as an HTTP header named **requesttoken**. The token is available in the global variable **oc_requesttoken**.
+If any other JavaScript request library than jQuery is being used, the requests need to send the CSRF token as an HTTP header named **requesttoken**. The token is available in the global variable **OC.requestToken**.
```
Comment (ChristophWurst) [inline developer_manual/basics/front-end/js.rst:37]: we should recommend https://nextcloud.github.io/nextcloud-auth/modules/_requesttoken_.html#getrequesttoken instead, but that requires a bundler. Not sure if that is too much for the tutorial.

Comment (juliushaertl) [reply developer_manual/basics/front-end/js.rst:37]: Yeah i think we could at least mention the package here, but for now I'd say having the OC.requestToken is still fine as a default example.

Comment (ChristophWurst) [reply developer_manual/basics/front-end/js.rst:37]: yeah, sure

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**